### DISCUSSION TITLE
Instant loading and custom Meta-Data header

### DISCUSSION THREAD

**Discussion by michael-nok** [squidfunk/mkdocs-material#3186] [Q&A] [answered]
I am experimenting with using customized YAML style meta-data header in my files as defined here:
https://www.mkdocs.org/user-guide/writing-your-docs/#yaml-style-meta-data

Sample YAML configuration:
```yaml
---
title: my title
description: my description
author: me!
date: 2021-11-04
foo: bar
---
```

When using instant loading, I can see that only a few meta data items are updated.  
Note: When instant loading is disabled, everything updates correctly, so this is my fall-back scenario.

https://github.com/squidfunk/mkdocs-material/blob/master/src/assets/javascripts/integrations/instant/index.ts
starting on line 275
```javascript
          /* Meta tags */
          "title",
          "link[rel=canonical]",
          "meta[name=author]",
          "meta[name=description]",
```

What would be the best way to expand the list of meta names being updated via instant loading?

```javascript
"meta[name=date]",
"meta[name=foo]",
```

**Comment by squidfunk** [comment, marked as answer]
Thanks for asking. You would need to patch them manually in the source code, since this is currently not officially supported.
